cdap-features/src/main/java/io/cdap/cdap/features/Feature.java

```diff
@@ -40,7 +40,8 @@ public enum Feature {
   SOURCE_CONTROL_MANAGEMENT_GIT("6.9.0"),
   WRANGLER_PRECONDITION_SQL("6.9.1"),
   WRANGLER_EXECUTION_SQL("6.10.0"),
-  WRANGLER_SCHEMA_MANAGEMENT("6.10.0");
+  WRANGLER_SCHEMA_MANAGEMENT("6.10.0"),
+  NAMESPACE_RESOURCE_ISOLATION("6.10.0");
```

Comment:
nit: Maybe we can change this to `NAMESPACED_SERVICE_ACCOUNTS` instead for accuracy?

Reply:
done.